### DISCUSSION TITLE
Modal 组件能能够自定义Header是不是更香？

### DISCUSSION THREAD

**Discussion by jkiss** [ant-design/ant-design#36903] [Ideas]
Modal组件现在header有一定的局限性，比如我想在右上角放2个icon，在左上角title位置放1个icon，就很难了，既然有自定义footer，何不再来个自定义header？

**Comment by MachoManCk** [comment]
多看些文档

**Reply by jkiss** [reply]
只看到了footer，没看到header呢

**Comment by izhouteng** [comment]
你为啥要在右上角放2个icon？
听起来就影响用户体验。
没必要。
国外的Modal组件也没有这样的。

**Reply by jkiss** [reply]
但我不是需求方啊😭

**Reply by izhouteng** [reply]
> 但我不是需求方啊😭

非要的话其实你可以自己封装下的

**Reply by jkiss** [reply]
已经用 `modalRender` 自定义了 😭
